src/models/network.js

```diff
@@ -138,27 +138,35 @@ Network.prototype.validate = function (client) {
 	}
 
 	if (Helper.config.lockNetwork) {
+		// Get the first configured network that matches this one, if any.
+		let defaultNetwork = Helper.getDefaultNetworks().filter(
```

Comment:
Can we store `getDefaultNetworks` so we don't print the `[WARN] Key "defaults" should now be` multiple times if they're using an old config?

Testing showed me

```sh
2022-04-20 00:32:40 [WARN] Key "defaults" should now be an array of networks. Support for the old object format will be removed in a future version, please update your config. https://thelounge.chat/docs/configuration#default-networks
2022-04-20 00:32:46 [WARN] Key "defaults" should now be an array of networks. Support for the old object format will be removed in a future version, please update your config. https://thelounge.chat/docs/configuration#default-networks
2022-04-20 00:32:48 [WARN] Key "defaults" should now be an array of networks. Support for the old object format will be removed in a future version, please update your config. https://thelounge.chat/docs/configuration#default-networks
2022-04-20 00:32:52 [WARN] Key "defaults" should now be an array of networks. Support for the old object format will be removed in a future version, please update your config. https://thelounge.chat/docs/configuration#default-networks
```